### ISSUE TITLE
I think "iOS 9 or later" should be changed to "iOS 13 or later".

### ISSUE THREAD

**Issue by LeeChangki** [aws-amplify/aws-sdk-ios#4986]
**State your question**
If the following statement is true, 
===
Note: v2 of Amplify Library for Swift (currently GA) is built on top of the [AWS SDK for Swift](https://github.com/awslabs/aws-sdk-swift) and only provides access to the AWS SDK for Swift which is currently in developer preview. You can access this underlying SDK via the Escape Hatch from AWS Amplify.
===
I think the sdk support as bellow:
===
The AWS SDK for Swift supports the following:
Swift 5.7 or higher
iOS & iPadOS 13.0 or higher
macOS 10.15 or higher
Ubuntu Linux 16.04 LTS or higher
Amazon Linux 2 or higher
===

Did I misunderstand?

**Comment by ruisebas** [comment]
Hi @LeeChangki ! Could you please clarify what is your question?

This is the repo for the **AWS SDK for iOS**, which supports iOS 9 and later. This SDK is used by Amplify **v1** only.

The repo for the **AWS SDK for Swift** (the one used by Amplify **v2**) is https://github.com/awslabs/aws-sdk-swift, and what they state matches what you've posted:
> The AWS SDK for Swift supports the following:
> 
> - Swift 5.7 or higher
> - iOS & iPadOS 13.0 or higher
> - macOS 10.15 or higher
> - Ubuntu Linux 16.04 LTS or higher
> - Amazon Linux 2 or higher
> Other environments (watchOS, tvOS, Windows, or others) may work but have not been verified.
> 
> These supported versions may change in the future.

**Comment by LeeChangki** [comment]
Hi @ruisebas , Thank you for quick response.
I thought this repo is for Amplify v2 because the release note is said 'version 2' https://github.com/aws-amplify/aws-sdk-ios/releases?page=20
Anyway, in summary,
Amplify v1 repo is https://github.com/aws-amplify/aws-sdk-ios.
Amplify v2 repo is https://github.com/aws-amplify/amplify-swift.
The base repo for Amplify v1 is https://github.com/amazon-archives/aws-sdk-ios-v1
The base repo for Amplify v2 is https://github.com/awslabs/aws-sdk-swift
Can I understand it as above?

**Comment by ruisebas** [comment]
Oh, I see the confusion now. Amplify and the AWS SDKs are actually different products, and as such they have their own versioning. 
Amplify internally uses the SDKs to interact with AWS services, with v1 using **AWS SDK for iOS** and v2 using **AWS SDK for Swift**.

The repo for both versions of Amplify is https://github.com/aws-amplify/amplify-swift, the `main` branch being used for v2 and the `v1` branch being used for v1.

To summarize:
- Amplify v2: https://github.com/aws-amplify/amplify-swift/tree/main
- AWS SDK for Swift: https://github.com/awslabs/aws-sdk-swift
- Amplify V1: https://github.com/aws-amplify/amplify-swift/tree/v1
- AWS SDK for iOS: https://github.com/aws-amplify/aws-sdk-ios

Let me know if this clarifies things and if you have other questions.

**Comment by LeeChangki** [comment]
Thank you for organizing it. It was helpful.
